common/src/test/java/bio/terra/catalog/service/DatasetServiceTest.java

```diff
@@ -132,11 +137,30 @@ void getMetadata() throws Exception {
     when(externalSystemService.getRole(SOURCE_ID)).thenReturn(DatasetAccessLevel.DISCOVERER);
     when(externalSystemService.getDataset(SOURCE_ID))
         .thenReturn(new StorageSystemInformation(DatasetAccessLevel.DISCOVERER));
-    when(datasetDao.retrieve(dataset.id())).thenReturn(dataset);
+    mockDataset();
     JSONAssert.assertEquals(
         metadataWithId(dataset.id()), datasetService.getMetadata(dataset.id()), true);
   }
 
+  @Test
+  void getMetadataInvalidUser() {
```

Comment:
This test wasn't changed but I wanted to organize all the `getMetadata()` tests together. Also our tests follow the convention that a test has the same name as the API it's testing, so I renamed it too.